frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/index.tsx

```diff
@@ -25,30 +23,36 @@ import {
   checkIfCancelled,
   checkIfFailed,
   checkIfLoading,
+  checkIfMetadataLoading,
```

Comment:
adding more hooks and util functions to handle checking conversion and polling the latest Collection